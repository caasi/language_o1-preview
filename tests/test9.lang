fun multiply x, y = x * y;
multiply(5);