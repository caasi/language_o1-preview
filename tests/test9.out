Error: Function 'multiply' expects 2 arguments but got 1
